### ISSUE TITLE
[NEWS] Przekazuj poprawnie adresy mail, by koszty przesyłki zostały prawidłowo rozliczone / Handle email addresses correctly, to ensure proper settlement for the shipping costs

### ISSUE THREAD

**Issue by TomekKaczmarek** [allegro/allegro-api#2121]
**Zmieniamy sposób rozliczania opłat za dostawę w ramach Allegro Smart!**. Sprzedający nie będzie już otrzymywał pieniędzy za przesyłkę realizowaną w ramach Allegro Smart!. Otrzyma wpłatę wyłącznie za sprzedany przedmiot.

Aby nadać przesyłkę bezpłatnie, w narzędziu nadawczym przewoźnika sprzedający wprowadza maskowany adres e-mail klienta dokładnie w takiej formie, jaką otrzymuje w zamówieniu. Na etykietach nadawczych adres e-mail klienta musi być pełny - powinien zawierać również ciąg znaków widoczny za “+” - np.: 8awgqyk6a5+**cub31c122**@allegromail.pl.

**Ważne!** Upewnij się, że twój system do zarządzania sprzedażą przekazuje pełny maskowany adres email dla każdego zamówienia.

Jeśli sprzedawca nada paczkę, ale **nie wprowadzi pełnego, maskowanego adresu e-mail klienta w domenie @allegromail.pl**, wysyłka nie będzie darmowa. Przewoźnik poinformuje o tym w Menedżerze Paczek, a na koniec miesiąca za dostawę tych paczek wystawi sprzedającemu fakturę.

**Kiedy wprowadzimy tę zmianę**
Zmianę zaczniemy wprowadzać stopniowo dla kolejnych sprzedawców od pierwszego tygodnia września 2019 r. Nowy model rozliczeń wprowadzimy dla wszystkich sprzedających 1 listopada 2019 r.

Więcej informacji na ten temat znajdziesz w [pomocy Allegro](https://allegro.pl/pomoc/artykul/allegro-smart-zmiana-sposobu-rozliczania-kosztow-przesylek-inpost-pomoc-i-kontakt-5V9rdLDOYf6?utm_source=newsletter).

---------------------------------------------------------------------------------------------

**We are going to change the approach to settlement of the shipping cost of Allegro Smart!** deliveries. Sellers will not receive the repayment for the Allegro Smart! delivery. They will be provided with payment for a product only.

To dispatch the delivery without any cost, sellers must transfer identical masked email address as in the order details provided by Allegro in the carrier Parcel Manager. The customer's e-mail address provided on a dispatch label must be complete - it should contain the string visible after "+" - e.g .: 8awgqyk6a5+**cub31c122**@allegromail.pl.

**Note!** Make sure your sales management system provides the full masked email address for each order

If a seller sends a package but **does not provide the customer's full masked email address in the @allegromail.pl** domain, wdelivery cost will be charged. The carrier will display a proper information in the Parcel Manager and at the end of the month he will issue an invoice to the seller for the delivery of these parcels.

**When we will introduce these changes**
The changes will be introduced gradually for subsequent sellers from the first week of September 2019. We will introduce the new billing model for all sellers on November 1, 2019.

You can find more details in [Allegro help](https://allegro.pl/pomoc/artykul/allegro-smart-zmiana-sposobu-rozliczania-kosztow-przesylek-inpost-pomoc-i-kontakt-5V9rdLDOYf6?utm_source=newsletter).

**Comment by jakubmaguza** [comment]
Zmiana dotyczy tylko Inpostu, uwzględnijcie to w treści, albo w tytule, bo brzmi, jakby dotyczyło całego smarta.
Czy planujecie podobną zmianę dla pozostałych przesyłek typu smart?

**Comment by PawelTaberski** [comment]
@jakubmaguza Obecnie zmiana dotyczy InPostu, jeśli lista zostanie poszerzona poinformujemy o tym w dedykowanym komunikacie.

**Comment by imper86** [comment]
@PawelTaberski czy klienci będą jakoś informowani o tym kiedy będą przenoszeni na nowy sposób rozliczenia? Rozstrzał czasowy od początku września do listopada to jest kupa czasu, a w najczarniejszym scenariuszu developerzy mają tylko 2 tygodnie na wprowadzenie zmian...

Mam nadzieję, że zdajecie sobie sprawę z tego, że tym razem to serio przegieliście z czasem na zastosowanie zmian, masakra...

**Comment by PawelTaberski** [comment]
@imper86 Jak najbardziej sprzedawcy zostaną odpowiednio mailowo powiadomieni o zmianie.

**Comment by mjag8** [comment]
@PawelTaberski 
Jaką wartość prześlecie w strukturze danych przy odpytaniu zasobu /order/checkout-forms dla zamówień łapiących się na zmianę?

Tj. co będzie w delivery.cost.amount ? Rozumiem, że zwrócicie 0.00?

**Comment by PawelTaberski** [comment]
Tak, będziemy w takim wypadku w tym polu zwracać zero.

**Comment by pijablonski** [comment]
Czy w polu delivery.cost.currency pozostanie w takim wypadku wartość "PLN", czy będą jakieś niespodzianki w stylu null lub innej wartości?

**Comment by TomekKaczmarek** [comment]
Zawsze powinniśmy zwracać poprawną walutę.

**Comment by jakubmaguza** [comment]
Pojawił się pewien problem - co z opłatą recyklingową?
Według ustawy muszę ją pobrać od klienta. Do tej pory ta kwota była wydzielona z wysyłki. Jeśli wysyłka będzie darmowa, to nie mogę wydzielić z niej opłaty recyklingowej. Niestety Allegro nie umożliwia dodania takiej opłaty do zamówienia.

**Comment by MaciejFrackowiak** [comment]
@jakubmaguza Proszę doprecyzuj nam swoje spostrzeżenie - dotychczas przykładowo dla przesyłki Allegro Paczkomaty opłata za przesyłkę wynosiła 8,99 zł i tyle maksymalnie płacił kupujący, a sprzedający taką kwotę uiszczał InPost za usługę. Wprowadzana zmiana obie kwoty po prostu zeruje.

**Comment by jakubmaguza** [comment]
Tak, tak to wyglądało, tylko biznes mógł dopłacić te 20 groszy do przesyłki z własnej kieszeni, a na fakturze rozbić przesyłkę na przykładowo: 8.79 - przesyłka + 0.20 opłata recyklingowa. Niestety ustawodawca zmusza do wpisania na fakturze opłaty recyklingowej.

**Comment by TimothyKoval** [comment]
To odejmij te 20gr od ceny towaru

**Comment by jakubmaguza** [comment]
Jest to pomysł który rozważamy, nie wiemy niestety jak zareagują klienci fakturowi jeśli na fakturze dostaną inną kwotę za produkt niż widzieli na Allegro.

**Comment by TimothyKoval** [comment]
Jeżeli opłata recyklingowa stanowi koszt uzyskania przychodu, to nie widzę problemu. Zawsze można dodać na aukcjach opis, że cena zawiera w/w opłatę.
